media/css/firefox/onboarding/firstrun-fxa.less

```diff
@@ -3,18 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 @import "../../sandstone/lib.less";
-
-.buildstar(@size, @left, @top, @speed, @delay) {
-    .animation(twinkle @speed ease-in-out @delay infinite alternate-reverse);
-    background: #fff;
-    border-radius: 50%;
-    height: @size;
-    left: @left;
-    opacity: 0;
-    position: absolute;
-    top: @top;
-    width: @size;
-}
+@import "onboarding-lib.less";
```

Comment:
Ah it's ok, on close inspection it does not seem to so perhaps this is indeed fine. Just seemed like a deviation from convention since it does generate some actual CSS.